sql/.sqlfluff

```diff
@@ -11,11 +11,12 @@ templater = jinja
 ## Comma separated list of rules to check, or None for all
 rules = None
 ## Comma separated list of rules to exclude, or None
-exclude_rules = L011,L014,L016,L020,L026,L027,L028,L029,L030,L031,L032,L034,L035,L036,L037,L042,L043,L051,L060
+exclude_rules = L011,L014,L016,L020,L022,L026,L027,L028,L029,L030,L031,L032,L034,L035,L036,L037,L042,L043,L051,L060
 # L011 - We don't always alias tables with AS ("FROM table1 AS tb1" instead of "FROM table1 tb1"). Do for columns but not for tables.
 # L014 - Unquoted identifiers (e.g. column names) will be mixed case so don't enforce case
 # L016 - We allow longer lines as some of our queries are complex. Maybe should limit in future?
 # L020 - Asks for unique table aliases meaning it complains if selecting from two 2021_07_01 tables as implicit alias is table name (not fully qualified) so same.
+# L022 - CTEs may be chained and do not require a blank line separator, only the last one.
```

Comment:
I guess it's a matter of personal preference and don't feel massively strongly about it.

But I think this:

```sql
WITH js_bytes AS (
  SELECT
    _TABLE_SUFFIX,
    url,
    bytesJs / 1024 AS kbytes_js
  FROM
    `httparchive.summary_pages.2022_06_01_*`
),

custom_elements AS (
  SELECT
    _TABLE_SUFFIX,
    url,
    COALESCE(ARRAY_LENGTH(JSON_VALUE_ARRAY(JSON_VALUE(payload, '$._wpt_bodies'), '$.web_components.rendered.customElements.names')) > 0, FALSE) AS has_custom_elements
  FROM
    `httparchive.pages.2022_06_01_*`
)
```

is easier to read than this:

```sql
WITH js_bytes AS (
  SELECT
    _TABLE_SUFFIX,
    url,
    bytesJs / 1024 AS kbytes_js
  FROM
    `httparchive.summary_pages.2022_06_01_*`
), custom_elements AS (
  SELECT
    _TABLE_SUFFIX,
    url,
    COALESCE(ARRAY_LENGTH(JSON_VALUE_ARRAY(JSON_VALUE(payload, '$._wpt_bodies'), '$.web_components.rendered.customElements.names')) > 0, FALSE) AS has_custom_elements
  FROM
    `httparchive.pages.2022_06_01_*`
)
```

And only gets worse with more complex queries.

Yes I get there's a comma in there so arguably part of the same statement, but they basically are as independent of each other as the CTEs are from the main query yet you argue for a line separator there, yet not between CTEs?

Reply:
Ok thanks. In that case if you don't have a strong preference, let's go ahead with disabling the rule. Resolving this, but feel free to reopen if you had any other thoughts.